.gitignore

```diff
@@ -134,4 +134,9 @@ out/
 
 # Quarkus specific
 .quarkus/
-quarkus.log
\ No newline at end of file
+quarkus.log
+
+# local personal templates 
+templates/personal/
+
+images/gemini-generation-image.png
```

Comment:
Thanks for this PR.
For the image generated, a new repertory as `build` or `result` could be interesting.